src/conftest.py

```diff
@@ -11,10 +13,19 @@
 import config
 from analysis.PluginBase import AnalysisBasePlugin
 from config import Config
-from test.common_helper import CommonDatabaseMock, create_docker_mount_base_dir
+from test.common_helper import CommonDatabaseMock
 from test.conftest import merge_markers
 
 
+def _create_docker_mount_base_dir() -> str:
+    dir = tempfile.mkdtemp(prefix='fact-docker-mount-base-dir')
+    docker_gid = grp.getgrnam('docker').gr_gid
+    os.chown(dir, -1, docker_gid)
+    os.chmod(dir, 0o770)
```

Comment:
This part seems to be mostly the same as what happens at the beginning of the `main()` method in `src/start_fact_backend.py`. Would it make sense to put it in a helper function (in fileSystem.py or docker.py maybe)?